addons/website/static/src/client_actions/configurator/configurator.xml

```diff
@@ -190,7 +190,7 @@
             <div class="m-auto w-100 w-md-75 w-xl-100">
                 <div class="o_configurator_typing_text text-center mt-4 mb-lg-4">Choose your favorite <b>Theme</b>
                 </div>
-                <div class="container">
+                <div class="container" dir="ltr">
```

Comment:
In fact, "click to select" is well translated into Arabic.. only the `theme_svg_container` should be forced into `ltr`... It's not a problem here because "click to select" is centered. But if we want to be correct, we should change that. As long as we're at it, wouldn't we rather put a rule in css with a small comment, what do you think?

Reply:
Yes indeed it's better to make the change on `theme_svg_container` only. But I would not do it in css because of this:
![image](https://github.com/odoo/odoo/assets/78849981/3c2c5c41-5c17-4e2b-8c7b-3a97ed732bb2)

I added an XML comments :+1: 

Let me know if it's ok for you now @bvr-odoo